### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by L33tHACOR** [openbullet/openbullet#319]
**Describe the bug**
I want to use JS to add up all the values of the list I made (full of integers). To see how the list from OB would look like in JS, I made it return a variable equal to that and it returned in ob as WITH VALUE System.Collections.Generic.List`1[System.String]

I tried using a simple loop to add up all the values, but that didn't work as I got INFO: No matching indexer found.
My List is called `a` with values [4, 3]
```
BEGIN SCRIPT JavaScript
var string = a;
var total = 0;
for (var i = 0; i < a.length; i++) {
total += a[i]
}
END SCRIPT -> VARS "total"
```
that is my code but it gives the no matching indexer found

This is in version 1.1.3#267

**Comment by openbullet** [comment]
Yes there is a bug I'm already aware of it. Will fix when I have time.

**Comment by openbullet** [comment]
https://github.com/openbullet/openbullet/commit/951caa03dab9415d2f9b8c7e2d9d64bbba6d31db

**Comment by bothudost371-pixel** [comment]
System.Collections.Generic.List<string>' to 'string'   when list of string is splited this this disturbs my mind
